pyproject.toml

```diff
@@ -77,7 +77,21 @@ poetry = "poetry.console:main"
 
 
 [build-system]
-requires = ["intreehooks"]
+requires = [
+    "intreehooks",
+    # According to PEP 517 we shouldn't rely on external dependencies.
```

Comment:
Should a comment be made above where we list the requirements to also update this list?